RubeesEat.UnitTests/Model/PersonTest.cs

```diff
@@ -0,0 +1,27 @@
+using RubeesEat.Model;
+
+namespace RubeesEat.UnitTests.Model;
+
+[TestFixture]
+public class PersonTest
+{
+    [Test]
+    public void Person_ValidParamenters_DoesNotThrowException()
+    {
+        Assert.DoesNotThrow(() => new Person(Guid.NewGuid(), "test", "test"));
+    }
+    
+    [Test]
+    [TestCase("Test2")]
+    [TestCase("3Test")]
+    [TestCase("?Test")]
+    [TestCase("1Test1")]
+    [TestCase("Te st")]
+    [TestCase(".")]
+    [TestCase("")]
+    [TestCase(" ")]
+    public void Person_NameInvalid_ThrowsException(string name)
```

Comment:
In BillTests you mentioned to write "Bill_ConstructorWithInvalidName_Throws.". Should we use the same convention here as well?

Reply:
I'm guessing we just have to be consistent. I will for now change it to

Bill_Constructor_WithSumNotZero_ThrowsException
Person_WithInvalidName_ThrowsException

We can change it if there is disagreement about this, but otherwise this should be consistent now.

Reply:
We agreed to
Bill_ConstructorWithSumNotZero_ThrowsException
Person_ConstructorWithInvalidName_ThrowsException

let us know if we need to change it